geth/signal/signals_test.go

```diff
@@ -0,0 +1,50 @@
+package signal
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNodeCrashEventJSONMarshalling(t *testing.T) {
```

Comment:
would be good to have all of these as a table tests if testify is not being used

Reply:
Shouldn't be necessary now that we'll only have one test for the NodeCrashEvent MarshalJSON.